app/models/game.rb

```diff
@@ -56,4 +48,144 @@ def populate_board!
   def piece_in_space?(x, y)
     pieces.where(position_x: x, position_y: y).any? ? true : false
   end
+
+  # set methods to define the kings and pieces of each color per DRY conventions
+  def black_king
+    King.find_by(game_id: id, type: 'King', color: 'black')
+  end
+
+  def white_king
+    King.find_by(game_id: id, type: 'King', color: 'white')
+  end
+
+  def black_pieces
+    Piece.where(game_id: id, color: 'black')
+  end
+
+  def white_pieces
+    Piece.where(game_id: id, color: 'white')
+  end
+  
+  # helper method for threatened_king?
+  def white_king_threatened?
+    white_king_x = white_king.position_x
+    white_king_y = white_king.position_y
+    # true if a black piece has a valid_move?(white_king_x, white_king_y)
+    black_pieces.each do |piece|
+      if piece.valid_move?(white_king_x, white_king_y)
+        return true
+      end
+    end
+    return false
+  end
+
+  # helper method for threatened_king?
+  def black_king_threatened?
+    black_king_x = black_king.position_x
+    black_king_y = black_king.position_y
+    # true if a white piece has a valid_move?(black_king_x, black_king_y)
+    white_pieces.each do |piece|
+      if piece.valid_move?(black_king_x, black_king_y)
+        return true
+      end
+    end
+    return false
+  end
+
+  # check? condition: CHECK FOR THREATENED KINGS
+  def threatened_king?
+    # if there is any piece whose valid_move? method returns true to a space with a King in it, return true
+    if white_king_threatened? || black_king_threatened?
+      return true
+    else
+      return false
+    end
+  end
```

Comment:
This doesn't need to be changed at all, just a suggustion if you like this way better. This method could be simplified to: 

`def threatened_king?
  return white_king_threatened? || black_king_threatened?
end`